lib/templates.ts

```diff
@@ -419,19 +421,25 @@ const _parseTemplate = (template: {
   id: string;
   jsonConfig: Prisma.JsonValue;
   isPublished: boolean;
+  updated_at?: Date;
```

Comment:
With the addition of the new `update_at` and `created_at` properties, you will need to make sure that whenever the `_parseTemplate` function is called the prisma query has to get those values. It is either that or we set both values to `n/a` (line 441, 442) if they are undefined.

Reply:
Thanks for taking a look. That makes sense about a check vs. having props with undefined/NaN.. Sure done. I went with the spread idea (from Bryan), probably the most obvious about what's going on.